test/cli_spec.js

```diff
@@ -10,13 +10,15 @@ import { expect } from 'chai';
 import installDeps from '../src/util/install-dependencies';
 
 const pSpawn = async (args = [], opts = {
-  stdio: 'inherit',
+  stdio: process.platform === 'win32' ? 'inherit' : 'pipe',
```

Comment:
@MarshallOfSound do you remember why this is configured this way? I think it's causing https://github.com/electron-userland/electron-forge-templates/issues/51